pkg/controller/klusterletaddon/klusterlet_addon_controller.go

```diff
@@ -289,6 +296,12 @@ func (r *ReconcileKlusterletAddon) Reconcile(request reconcile.Request) (reconci
 		return reconcile.Result{}, err
 	}
 
+	//update managedcluster label for PolicyController
```

Comment:
add a comment here that it  is a  workaround and will be  removed when placement supports it.